pkg/model/command.proto

```diff
@@ -61,9 +61,10 @@ message Command {
     string id = 1 [(validate.rules).string.min_len = 1];
     string piped_id = 2 [(validate.rules).string.min_len = 1];
     string application_id = 3;
-    string deployment_id = 4;
-    string stage_id = 5;
-    string commander = 6;
+    string project_id = 4;
+    string deployment_id = 5;
+    string stage_id = 6;
+    string commander = 7;
```

Comment:
We have to keep the field numbers to ensure backward-compatibility.
The data with the old structure may be still in the cache or somewhere.

Reply:
Nice catch 👍 And besides, should it be `string project_id = 7;` 👀

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/1772/commits/df9a46b82333b2059064f9b8b42fac038c4406a4 🙏